source/buildings/restaurant.md

```diff
@@ -49,7 +49,7 @@ This is page two of the Restaurant GUI.
   </div>
   <div class="col-sm-12 col-md">
       <ul>
-      <li><strong> Fuel: </strong> Listed here are items that can be used by the Cook as fuel in their furnaces. Simply turn on any that you want your Cook to use, and the Deliveryman will deliver those items to the Cook when they need fuel. (The black box at the top is to search for items.)
+      <li><strong> Fuel: </strong> Listed here are items that can be used by the Cook as fuel in their furnaces. Simply turn on any that you want your Cook to use, and the <a href="https://wiki.minecolonies.ldtteam.com/source/workers/courier">Courier</a> will deliver those items to the Cook when they need fuel. (The black box at the top is to search for items.)</li>
```

Comment:
You can just do <a href="../../source/workers/courier"> Courier</a> to save space